catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterDelegate.java

```diff
@@ -130,104 +130,74 @@ public FilterType propertyIsEqualTo(String propertyName, String literal,
             boolean isCaseSensitive) {
         isComparisonOperationSupported(ComparisonOperatorType.EQUAL_TO);
         propertyName = mapPropertyName(propertyName);
-        if (isPropertyQueryable(propertyName)) {
-            return cswFilterFactory.buildPropertyIsEqualToFilter(propertyName,
-                    literal,
-                    isCaseSensitive);
-        } else {
-            return new FilterType();
-        }
+        return cswFilterFactory.buildPropertyIsEqualToFilter(propertyName,
+                literal,
+                isCaseSensitive);
     }
 
     @Override
     public FilterType propertyIsEqualTo(String propertyName, Date literal) {
         isComparisonOperationSupported(ComparisonOperatorType.EQUAL_TO);
         propertyName = mapPropertyName(propertyName);
-        if (isPropertyQueryable(propertyName)) {
-            return cswFilterFactory.buildPropertyIsEqualToFilter(propertyName,
-                    convertDateToIso8601Format(literal),
-                    false);
-        } else {
-            return new FilterType();
-        }
+        return cswFilterFactory.buildPropertyIsEqualToFilter(propertyName,
+                convertDateToIso8601Format(literal),
+                false);
+
     }
 
     @Override
     public FilterType propertyIsEqualTo(String propertyName, int literal) {
```

Comment:
Now that these methods are pretty identical it may be worth it to have AbstractCswFilterDelegate extend the SimpleFilterDelegate instead of the FilterDelegate. Then you only would need to override the base method and remove all this code duplication.